packages/disputer/index.js

```diff
@@ -52,8 +52,13 @@ async function run({
     // Setup web3 accounts and network
     const [accounts, networkId] = await Promise.all([web3.eth.getAccounts(), web3.eth.net.getId()]);
 
-    // Setup contract instances. NOTE that getAddress("Voting", networkId) will resolve to null in tests.
-    const voting = new web3.eth.Contract(getAbi("Voting"), getAddress("Voting", networkId));
+    // Setup contract instances. NOTE getAddress("Voting", networkId) will error if run in tests. Catch & set it to null.
```

Comment:
Previously voting would simply return `null` when running the `index.js` unit tests for the monitor, liquidator and disputer. However, when run with hardhat, this throws an error that breaks the tests. The easiest solution is to just try & catch the response of this call.